core/src/modules/ops/profile.ts

```diff
@@ -729,61 +724,37 @@ export namespace ProfileOps {
   export async function confirmExistence(
     limit: number,
     fromDate: Date,
-    sourceId?: string,
-    run?: Run
+    sourceId?: string
   ) {
     const properties = await Property.findAllWithCache();
     const directlyMapped = properties.filter(
       (p) => p.directlyMapped && (!sourceId || sourceId === p.sourceId)
     );
 
-    const profiles = await Profile.findAll({
-      limit,
-      where: { state: "ready" },
-      include: [
-        GroupMember,
-        {
-          model: ProfileProperty,
-          required: true,
-          where: {
-            state: "ready",
-            confirmedAt: {
-              [Op.lt]: fromDate,
-            },
-            rawValue: {
-              [Op.ne]: null,
-            },
-            propertyId: directlyMapped.map((p) => p.id),
-          },
+    const profileProperties = await ProfileProperty.findAll({
+      where: {
+        state: "ready",
+        confirmedAt: {
+          [Op.lt]: fromDate,
         },
-      ],
+        rawValue: {
+          [Op.ne]: null,
+        },
+        propertyId: directlyMapped.map((p) => p.id),
+      },
+      limit,
     });
 
-    const now = new Date();
-    const bulkImports = [];
-
-    const creatorInfo = run
-      ? { creatorType: "run", creatorId: run.id }
-      : { creatorType: "task", creatorId: "profiles:confirm" };
-
-    for (const profile of profiles) {
-      delete profile.profileProperties; // get all profile properties
-      const oldProfileProperties = await profile.simplifiedProperties();
-      const oldGroupIds = profile.groupMembers.map((gm) => gm.groupId);
-
-      bulkImports.push({
-        profileId: profile.id,
-        profileAssociatedAt: now,
-        oldProfileProperties,
-        oldGroupIds,
-        ...creatorInfo,
-      });
-    }
+    const profileIds = profileProperties.map((pp) => pp.profileId);
 
-    await Import.bulkCreate(bulkImports);
-    await markPendingByIds(profiles.map((p) => p.id));
+    // Only mark profile and directlyMapped property pending
+    await markPendingByIds(profileIds, false);
+    await ProfileProperty.update(
+      { state: "pending", startedAt: null },
+      { where: { id: profileProperties.map((pp) => pp.id) } }
+    );
 
-    return profiles.length;
+    return profileProperties.length;
```

Comment:
The return here used to be `profiles.length` - this is equivalent now only if there is one directlyMapper property.  Perhaps `const profileIds = profileProperties.map((pp) => pp.profileId);` above should be uniqe'd and then counted here?